PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -122,11 +122,11 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
             //pass along the command so the char is added to the buffer 
             int retVal = NextCommandHandler.Exec(ref pguidCmdGroup, nCmdId, nCmdexecopt, pvaIn, pvaOut);
             bool handled = false;
-            if (!typedChar.Equals(char.MinValue) && IsIntellisenseTrigger(typedChar))
+            if (!typedChar.Equals(Char.MinValue) && IsIntellisenseTrigger(typedChar))
             {
                 TriggerCompletion();
             }
-            if (!typedChar.Equals(char.MinValue) && IsFilterTrigger(typedChar))
+            if (!typedChar.Equals(Char.MinValue) && IsFilterTrigger(typedChar))

```

Comment:
.NET Coding Guidelines recommend we use the language-helper, instead of the class itself.